src/LiveDevelopment/MultiBrowserImpl/protocol/remote/LiveDevProtocolRemote.js

```diff
@@ -272,7 +272,6 @@
         },
         
         onClose: function () {
-            // TODO: This is absolutely temporary solution.
```

Comment:
@le717 :) it's been reworked from its original version, but the comment was left. what we have now seems to be satisfactory.